modules/ROOT/pages/viewing-log-data.adoc

```diff
@@ -2,53 +2,40 @@
-image:logo-pcf-disabled.png[link="/runtime-manager/deployment-strategies", title="Pivotal Cloud Foundry"]
 
 CloudHub provides access to log data that includes deployment messages and events for each worker.
 
-
 [NOTE]
 ====
-Logs and diagnostics are currently available only for applications deployed to CloudHub workers. In cases where applications are deployed using the Runtime Manager local server, you can send data xref:sending-data-from-arm-to-external-analytics-software.adoc[to External Analytics Software]. See xref:deployment-strategies.adoc[Deployment Strategies] and xref:monitoring.adoc[Monitoring Applications] for more details.
+Logs and diagnostics are currently available only for applications deployed to CloudHub workers. In cases where applications are deployed to a local server, you can send data to external analytics software. 
 
 Applications deployed to Pivotal Cloud Foundry (PCF) don't expose any log data through Runtime Manager, but you can see event logs on Pivotal's UI.
 ====
 
 
 == View Logs
 
-. In Runtime Manager, click *Applications* in the left menu, then click the application that is deployed to CloudHub to open its management options.
-. In the left menu, click *Logs*. The application logs are displayed.
-+
-//image::elmstart.png[ELMStart]
+View logs for applications deployed to CloudHub.
 
+. In Runtime Manager, click *Applications* in the left menu, then click the application that is deployed to CloudHub to open the management panel on the right. 
+. Click *Logs*. The application logs are displayed.
 . The *Deployments* section on the right of the logging screen lets you selectively view the output by day, deployment, system log, and workers. You can unapply a filter by unticking the checkbox next to a date.
 +
 image::elmfilters.png[ELMfilters]
 
-. You can search for log content, or click *Advanced* to search for a specified time, day, date range, or by message priority. After you specify the search criteria, click *Apply*. For more information, see <<Advanced Search, Advanced Search>>.
-+
-image::elmadvancedsearch.png[ELMAdvancedSearch]
-
-
-
 == Download Logs
 
-Each worker in each deployment has separate logs you can download.
+Each worker in each CloudHub deployment has separate logs you can download.
 
-In the *Deployment* filters section on the right, find the date and worker you want to view, click the download icon next to it, and select *Logs*.
+. In the *Deployment* filters section on the right, find the date and worker you want to view. 
+. Click the download icon next to the worker, and select *Logs*.
 
 image::downloadlogs.png[DownloadLogs]
 
-The log file is a text file with `.log` file extension and named in the format of `<deployment-ID>-<instance-ID>.log`. You can view this file with a text editor.
+The log file is a text file with `.log` file extension and named in the format of `<deployment-ID>-<instance-ID>.log`. You can view the log file with a text editor.
```

Comment:
The log file is a text file with the format '<deployment... .log'.

The format provides the extension. But if it is important to call that out, then STET with the following slight adjustments: "...with the file extention .log, in the format '<deployment...."